tests/bulk_create/tests.py

```diff
@@ -35,21 +38,39 @@ def test_efficiency(self):
         with self.assertNumQueries(1):
             Country.objects.bulk_create(self.data)
 
-    def test_inheritance(self):
-        Restaurant.objects.bulk_create([
-            Restaurant(name="Nicholas's")
-        ])
-        self.assertQuerysetEqual(Restaurant.objects.all(), [
-            "Nicholas's",
-        ], attrgetter("name"))
-        with self.assertRaises(ValueError):
+    def test_multi_table_inheritance_unsupported(self):
+        expected_message = "Can't bulk create a multi-table inherited model"
+        with self.assertRaisesMessage(ValueError, expected_message):
             Pizzeria.objects.bulk_create([
                 Pizzeria(name="The Art of Pizza")
             ])
-        self.assertQuerysetEqual(Pizzeria.objects.all(), [])
-        self.assertQuerysetEqual(Restaurant.objects.all(), [
-            "Nicholas's",
-        ], attrgetter("name"))
+        with self.assertRaisesMessage(ValueError, expected_message):
+            ProxyMultiCountry.objects.bulk_create([
+                ProxyMultiCountry(name="Fillory", iso_two_letter="FL")
+            ])
+        with self.assertRaisesMessage(ValueError, expected_message):
+            ProxyMultiProxyCountry.objects.bulk_create([
+                ProxyMultiProxyCountry(name="Fillory", iso_two_letter="FL")
+            ])
+        expected_empty_models = (Pizzeria, Restaurant, Country, ProxyCountry,
```

Comment:
Yes. This test prevents a regression in which the ValueError gets moved in the code to after where the QuerySet tries to execute the write and can only get part way through it.

I think having this here is important, but ultimately it's your code base. Let me know if removing it is necessary to getting this merged.